deploy/olm-catalog/kogito-operator/0.13.0/app.kiegroup.org_kogitobuilds_crd.yaml

```diff
@@ -4,30 +4,30 @@ metadata:
   name: kogitobuilds.app.kiegroup.org
```

Comment:
still changed ?

Reply:
@vaibhavjainwiz please rollback changes in the `deploy/olm-catalog/kogito-operator/0.13.0` directory since we already released this version. Please run `make test` in the project's root to update the `1.0.0-snapshot` manifest. So when we branch to 0.14, everything will be in place. Thanks.

Reply:
Yes, I did same. But it autumatically adding some spaces which causing it to come in change list

Reply:
File comming in changelist due to auto formating issue with Intellij IDE. I have removed this file now